dotcom-rendering/src/web/components/FixedSmallSlowVHalf.tsx

```diff
@@ -0,0 +1,56 @@
+import type { DCRContainerPalette, DCRFrontCard } from '../../types/front';
+import { Card50Media50, CardDefaultMediaMobile } from '../lib/cardWrappers';
+import { LI } from './Card/components/LI';
+import { UL } from './Card/components/UL';
+
+type Props = {
+	trails: DCRFrontCard[];
+	containerPalette?: DCRContainerPalette;
+	showAge?: boolean;
+};
+
+export const FixedSmallSlowVHalf = ({
+	trails,
+	containerPalette,
+	showAge,
+}: Props) => {
+	const firstSlice50 = trails.slice(0, 1);
+	const remaining = trails.slice(1, 5);
+
+	return (
+		<UL direction="row-reverse">
```

Comment:
On desktop the 50% card is the last card hence row-reverse, but on mobile it switches to `column` and it needs to be the first card.